UWP/UWP.vcxproj.filters

```diff
@@ -490,6 +490,17 @@
     <None Include="Content\debugger\static\media\*.svg">
       <Filter>Content\debugger\static\media</Filter>
     </None>
+    <None Include="Content\debugger\static\css\*.css" />

```

Comment:
Intentional?  I think these were just removed?  I assume VS is adding them back automatically.

-[Unknown]

Reply:
yeah, presumably. If they're getting added back automatically anyway, I'm gonna leave them in.